src/sql/src/func.rs

```diff
@@ -1789,6 +1789,7 @@ lazy_static! {
             "date_trunc" => Scalar {
                 params!(String, Timestamp) => BinaryFunc::DateTruncTimestamp, 2020;
                 params!(String, TimestampTz) => BinaryFunc::DateTruncTimestampTz, 1217;
+                params!(String, Interval) => BinaryFunc::DateTruncInterval, 1172;
```

Comment:
Not sure about the OID, but shouldn't it be 1218? https://github.com/postgres/postgres/blob/master/src/include/catalog/pg_proc.dat#L2434-L2436

Reply:
yes